crates/router/src/routes/metrics.rs

```diff
@@ -57,6 +57,12 @@ counter_metric!(RESPONSE_DESERIALIZATION_FAILURE, GLOBAL_METER);
 counter_metric!(CONNECTOR_ERROR_RESPONSE_COUNT, GLOBAL_METER);
 counter_metric!(REQUEST_TIMEOUT_COUNT, GLOBAL_METER);
 
+counter_metric!(EXECUTE_PRETASK_COUNT, GLOBAL_METER);
+counter_metric!(CONNECTOR_PAYMENT_METHOD_TOKENIZATION, GLOBAL_METER);
+counter_metric!(PREPROCESSING_STEPS_COUNT, GLOBAL_METER);
```

Comment:
Will PRETASK/PREPROCESSING be meaningful when checked on a dashboard like grafana. Is it bundling many things? Should you add more attributes to differentiate?

Reply:
For PRETASK , I have added attribute for `flow` and for `PREPROCESSING`, I have added attribute for `payment_method` and `payment_method_type` along with `connector`.